tools/depends/target/libicundk/Makefile

```diff
@@ -0,0 +1,39 @@
+include ../../Makefile.include
+DEPS=../../Makefile.include Makefile ../../download-files.include
+
+# lib name, version
+LIBNAME=libicundk
+VERSION=master
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+BASE_URL=https://gitlab.com/koying/libicundk/-/archive/master
+SHA256=48C3694692661260A6CD8B480E0B9CF1AE09240EE5E772A68870B1054D0D9BAE
```

Comment:
you cannot link to the "master", since this will break once a new commit is made